.github/workflows/ci.yml

```diff
@@ -44,3 +44,18 @@ jobs:
         run: pnpm run test:unit
       - name: Run e2e tests
         run: pnpm run test:e2e
+
+  docker:
```

Comment:
I think it will be better if you do the test using `sh` directly since installing pnpm is an overhead